xmlencoder.go

```diff
@@ -4,23 +4,28 @@ import (
 	"encoding/xml"
 	"fmt"
 	"github.com/dasfoo/gmail-filters/filter"
-	"os"
+	"github.com/dasfoo/gmail-filters/writer"
 	"time"
 )
 
 func main() {
 
 	v := &filter.Feed{Title: "Mail Filters", Update: time.Now()}
-	v.AddAuthor("Artem Sheremet", "dotdoom@gmail.com")
+	v.AddAuthor("Ivan Ivanov", "ivan@mail.ru")
+	v.Xmlns = "http://www.w3.org/2005/Atom"
```

Comment:
This (as well as Xmlnapp) may be set in the NewFeed() constructor (btw, we need one)